pkg/reconciler/openshift/tektonconfig/testdata/postreconcile_manifest/test_post_manifest.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2023 The Tekton Authors
```

Comment:
```suggestion
# Copyright 2024 The Tekton Authors
```

Reply:
@savitaashture sure, I will update the year.
I started this PR on 2023 :smile:

Reply:
Oh :laughing: